### ISSUE TITLE
Enabling Bootstrap's transitions

### ISSUE THREAD

**Issue by chsreedhar** [TheCoder4eu/BootsFaces-OSP#993]
Add **`Bootstrap: transition.js`** as default to all views for simple transition effects.
[Transitions transition.js](https://getbootstrap.com/docs/3.3/javascript/#transitions)

**Comment by stephanrauh** [comment]
I didn't implement this yet, but I think it's a good idea.

I guess simply adding the library isn't enough. Can you give me a hint which components benefit from the `fade` CSS class? Are there other benefits of adding `transition.js`?

**Comment by chsreedhar** [comment]
A few examples of the transition plugin:

Sliding or fading in modals
Fading out tabs
Fading out alerts
Sliding carousel panes

**Comment by TheTimeWalker** [comment]
Actually from initial testing, simply adding the library seems to work out pretty well! I've used this file from v3.3.4 to test: https://raw.githubusercontent.com/twbs/bootstrap/master/js/transition.js
The only thing that you have to do is adding a class `fade` to stuff like modals, tab panes, etc. and it works out of the box

**Comment by stephanrauh** [comment]
Good point. Adding the transition.js in your application is no big deal. Maybe we'll add them to the core library nonetheless because the default distribution of Bootstrap also includes transition.js.
